Dockerfile

```diff
@@ -10,16 +10,16 @@ RUN groupadd --gid 1000 node \
   && useradd --uid 1000 --gid node --shell /bin/bash --create-home node \
   && ARCH= && dpkgArch="$(dpkg --print-architecture)" \
   && case "${dpkgArch##*-}" in \
-  amd64) ARCH='x64';; \
-  ppc64el) ARCH='ppc64le';; \
-  s390x) ARCH='s390x';; \
-  arm64) ARCH='arm64';; \
-  armhf) ARCH='armv7l';; \
-  i386) ARCH='x86';; \
-  *) echo "unsupported architecture"; exit 1 ;; \
+    amd64) ARCH='x64';; \
+    ppc64el) ARCH='ppc64le';; \
+    s390x) ARCH='s390x';; \
+    arm64) ARCH='arm64';; \
+    armhf) ARCH='armv7l';; \
+    i386) ARCH='x86';; \
+    *) echo "unsupported architecture"; exit 1 ;; \
   esac \
   && set -ex \
-  && apt-get update && apt-get install -y zlib1g=1:1.2.11.dfsg-1+deb10u1 ca-certificates curl wget gnupg dirmngr xz-utils --no-install-recommends \
+  && apt-get update && apt-get install -y ca-certificates curl wget gnupg dirmngr xz-utils --no-install-recommends \
```

Comment:
Are we missing something here? Where do we install the `zlib` patch?

Reply:
It may not be needed in Debian bullseye. Once the image is built I can feed it to snyk for scanning.